ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -104,8 +104,13 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
     final Optional<List<Withdrawal>> maybeWithdrawals =
         Optional.ofNullable(blockParam.getWithdrawals())
             .map(ws -> ws.stream().map(WithdrawalParameter::toWithdrawal).collect(toList()));
-    if (!getWithdrawalsValidator(timestampSchedule, blockParam.getTimestamp())
-        .validateWithdrawals(maybeWithdrawals)) {
+    final Optional<BlockHeader> maybeParentHeader =
+        protocolContext.getBlockchain().getBlockHeader(blockParam.getParentHash());
+    boolean ableToValidateWithdrawals = maybeParentHeader.isPresent();
+    if (ableToValidateWithdrawals
```

Comment:
This does have a side-effect change of behaviour such that for an "invalid withdrawals"-block, we will end up returning SYNCING if the parent header is missing. Before this PR, we would return an "Invalid params" error, however that may be incorrect - I am waiting fir clarification about the spec w.r.t syncing.

Reply:
I got some clarification on syncing...on the one hand it makes sense to defer SYNCING until after we've performed all possible validation, bearing in mind not all validation can take place without the parent header.
On the other hand, there might be a performance trade-off: if a CL is spamming us with requests then breaking out early to respond with SYNCING may help.

Performance aside, I think the proper order is to respond with INVALID, if not SYNCING, so currently thinking I shouldn't rely on the parent header being present to validate withdrawals